test/routes/actions.test.js

```diff
@@ -0,0 +1,61 @@
+// let actionService
+require('../../server/services/actionsService')
+const mockActionServiceResult = [
+  {
+    id: 'ID1',
+    description: 'Test',
+    rules: [{
+      id: 1,
+      description: 'Test rule',
+      enabled: true,
+      facts: []
+    }]
+  }
+]
+
+const mockActionsService = {
+  getActions: jest.fn().mockResolvedValue(mockActionServiceResult)
+}
+
+function createMocks () {
+  jest.mock('../../server/services/actionsService', () => mockActionsService)
+}
+
+const requestOptions = {
+  method: 'GET',
+  url: '/actions'
+}
+
+describe('Actions test', () => {
+  let createServer
+  let server
+
+  beforeAll(() => {
+    createMocks()
+    createServer = require('../../server/createServer')
+  })
+
+  beforeEach(async () => {
+    server = await createServer()
+    await server.initialize()
+  })
+
+  test('GET /actions route calls the actionsService', async () => {
+    await server.inject(requestOptions)
+    expect(mockActionsService.getActions).toHaveBeenCalled()
+  })
+
+  test('GET /actions route gets a 200 response', async () => {
+    const getResponse = await server.inject(requestOptions)
+    expect(getResponse.statusCode).toBe(200)
+  })
+
```

Comment:
Open question: should we have a test that checks the content type of the response?